src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java

```diff
@@ -304,31 +305,49 @@ private void parseQueryMeta(ResultSet rsQueryMeta) throws SQLServerException {
         }
     }
 
-    private void parseQueryMetaFor2008(ResultSet rsQueryMeta) throws SQLServerException {
-        ResultSetMetaData md;
+    private void parseFMTQueryMeta(ResultSetMetaData md, SQLServerFMTQuery f) throws SQLServerException {
         try {
-            if (null != rsQueryMeta) {
-                md = rsQueryMeta.getMetaData();
-
-                for (int i = 1; i <= md.getColumnCount(); i++) {
-                    QueryMeta qm = new QueryMeta();
-
-                    qm.parameterClassName = md.getColumnClassName(i);
-                    qm.parameterType = md.getColumnType(i);
-                    qm.parameterTypeName = md.getColumnTypeName(i);
-                    qm.precision = md.getPrecision(i);
-                    qm.scale = md.getScale(i);
-                    qm.isNullable = md.isNullable(i);
-                    qm.isSigned = md.isSigned(i);
-
-                    queryMetaMap.put(i, qm);
+            List<String> columns = f.getColumns().stream().collect(Collectors.toList());
```

Comment:
Since columns and f.getColumns both return List of type String, do we need to do `.stream().collect(Collectors.toList())`?

Reply:
No we do not, will fix.